app/src/lib/globals.d.ts

```diff
@@ -41,57 +41,6 @@ declare const __UPDATES_URL__: string
  */
 declare const __PROCESS_KIND__: 'main' | 'ui' | 'crash' | 'highlighter'
 
-/**
```

Comment:
With upgrade in typescript, do these things just exist and is why we don't need them in globals.d.ts anymore?

Reply:
Yup!

It seems like TypeScript goes a bit behind the latest API standard, and updates their [lib.dom.d.ts](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts) later than we get new features from Electron/Chromium updates.